### PR TITLE
Merge Home 'Do first' and 'To-do list' cards into one compact priority card

### LINKED ISSUE #4095: Merge Home 'Do first' and 'To-do list' cards into one compact priority card
## Investigation

`apps/app/src/pages/Home.tsx` renders two overlapping surfaces from the same data source (`home.actionItems`):

1. **`TodayPriorityCard`** ("Do first", ~line 695/827): shows `topAction = home.actionItems[0]`, plus a footer with Open action / Ask AI / Next up.
2. **"To-do list" section** (~line 748–782): shows `remainingActions` (items 1–6 of the same array), a count badge, `MultiRsvpLink` when `rsvpNeeded > 1`, and "Priority shown above" / "All caught up" empty states.

Both cards present the same to-do queue split across two surfaces; the second card's dominant empty state ("Priority shown above") exists only to explain the split. Merging removes redundant UI and vertical scroll.

## Requirements (EARS)

1. WHEN a parent has one or more open actions, THE home page SHALL show a single priority card containing the top action (title, detail, icon) AND up to 3 remaining actions as compact rows, with an overflow count for the rest.
2. WHEN a parent has zero open actions, THE home page SHALL show the existing "All caught up" state in that single card only (no second card).
3. WHEN details are loading, THE card SHALL show the existing loading state only once.
4. WHEN `rsvpNeeded > 1`, THE card SHALL retain the multi-RSVP shortcut.
5. THE footer (Open action / Ask AI / Next up) SHALL be preserved unchanged.
6. THE standalone "To-do list" section SHALL be removed, including its "Priority shown above" state.
7. Existing analytics triggers (`priority_action`, action-row drill-ins) SHALL be preserved.

## Design

- Extend `TodayPriorityCard` props with `remainingActions: ParentHomeAction[]`, `remainingActionCount: number`, and `multiRsvpTo: string | null`.
- Inside the card, after the top-action block and before `PriorityFooter`, render `remainingActions.slice(0, 3)` as existing `ActionRow`s in a bordered list, header row "Also to do · N" with count badge and optional `MultiRsvpLink`.
- Delete the `home-action-section` block; `remainingActions`/`remainingActionCount` computation stays in the parent and is passed down (list capped at 3 instead of 6).
- Empty/loading/caught-up branches unchanged, single-sourced in the card.

## Tasks

- [ ] 1. Update `TodayPriorityCard` to accept and render remaining actions + count + multi-RSVP link (Req 1, 4, 5)
- [ ] 2. Remove the standalone To-do list section (Req 2, 6)
- [ ] 3. Update `Home.test.tsx`: adjust the lone-action test (no more "Priority shown above"), add a test that remaining actions render inside the priority card with overflow count (Req 1–3, 7)
- [ ] 4. Run app unit tests + typecheck + build

## Verification

- `cd apps/app && npx vitest run src/pages/Home.test.tsx`
- `npm run app:build` boots and the home route renders one card
- CI: unit-tests, app-quality, cache-bust-guard, preview-smoke green

### PR DESCRIPTION
**Pull request by pauljsnider**
Closes #4095

## Summary
- Fold the standalone "To-do list" section into the "Do first" priority card: remaining actions render as up to 3 compact rows under the top action with an overflow count
- Keep the multi-RSVP shortcut, count badge, analytics triggers, and the Open action / Ask AI / Next up footer unchanged
- Remove the duplicated empty states ("Priority shown above") so loading/caught-up copy exists once

## Manual test steps
1. `npm run app:dev`, sign in as a parent with open actions
2. Home shows a single "Do first" card: top action + "Also to do" rows (max 3) + `+N more` overflow
3. With exactly one action: no "Also to do" block, no second card
4. With zero actions: single "All caught up" card
5. With `rsvpNeeded > 1`: Multi RSVP link appears in the "Also to do" header

## Tests
- Updated `Home.test.tsx` lone-action test; new test for remaining actions + overflow inside the priority card; adjusted large-payload proxies. 43/43 pass, `tsc --noEmit` clean, `app:build` + bundle budget green.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: apps/app/src/pages/Home.tsx [hunk 1]
```diff
@@ -885,6 +863,25 @@ function TodayPriorityCard({ action, nextEvent, loading }: { action: ParentHomeA
           <p className="mt-1 line-clamp-2 text-sm font-semibold leading-5 text-gray-600">{action.detail}</p>
         </div>
       </div>
+      {remainingActions.length ? (
```
[the remark this answers is not in the document]
Comment (pauljsnider) [reply apps/app/src/pages/Home.tsx:866]: Fixed in follow-up commit: the integration test now asserts the removed copy is absent (tests/unit/app-home-player-integration.test.jsx:580).

**Comment by pauljsnider** [comment]
Reviewed current head `262ee09384fe`.

paulbot-current-head-review: no blocking findings

│
◇  Doctor warnings ───────────────────────────────────────────────────────────╮
│                                                                             │
│  - Left 3 Codex binding sidecar(s) in place because session ownership       │
│    is indeterminate: session index                                          │
│    /home/paul-bot1/.openclaw-codex-core/agents/main/sessions/sessions.json  │
│    has invalid entries                                                      │
│                                                                             │
├─────────────────────────────────────────────────────────────────────────────╯
paulbot-current-head-review: no blocking findings

Reviewed the Home card merge, action counts/overflow, navigation and analytics reuse, loading/empty states, multi-RSVP behavior, and focused test coverage. Changes are scoped to the PR intent with no P0–P2 regression identified. Local test execution was unavailable because dependencies were not installed.
